code/modules/cargo/packs.dm

```diff
@@ -1451,8 +1451,37 @@
 					/obj/item/vending_refill/medical)
 	crate_name = "medical supplies crate"
 
-/datum/supply_pack/medical/supplies/fill(obj/structure/closet/crate/C)
```

Comment:
Am i losing it or is yhis just gone

Reply:
....it might be let me check

Reply:
/datum/supply_pack/medical/supplies/fill(obj/structure/closet/crate/C)
	for(var/i in 1 to 10)
		var/item = pick(contains)
		new item(C) ?

still there

Reply:
you were right lol